etc/config/glsl.defaults.properties

```diff
@@ -0,0 +1,12 @@
+compilers=&glslang
+
+defaultCompiler=glslangValidator
+supportsBinary=false
+compilerType=glsl
+instructionSet=spirv
+
+disassemblerPath=/opt/compiler-explorer/SPIRV-Tools-master/build/tools/spirv-dis
+
+group.glslang.compilers=glslangValidator
+compiler.glslangValidator.exe=/home/fricke/1.3.290.0/x86_64/bin/glslangValidator
```

Comment:
Should be something default like when you `cmake --install` for example or install from `apt-get` or similar. (`/usr/bin/glslangValidator` maybe?)

Reply:
I guess not sure if `glslang` has an `apt-get` packaging as I only ever get it from the Vulkan SDK ... (this was clearly for testing, I was hoping someone tell me what it should actually should be for the checked in version, so I guess `/usr/bin/glslangValidator` is better then nothing)